nengo_gui/static/components/netgraph_item.js

```diff
@@ -356,8 +356,11 @@ Nengo.NetGraphItem.prototype.generate_menu = function () {
         }
     }
     if (this.sp_targets.length > 0) {
-        items.push(['Semantic pointer',
+        // Naming things is hard
```

Comment:
Yes, one of the two things that are hard in computer science. The other:
1. cache invalidation
2. off-by-one errors

Fun aside: This comment doesn't really help.